Added Introduction Video (#7788)
kamranahmedse/developer-roadmap: src/data/roadmaps/ai-engineer/content/introduction@_hYN0gEi9BL24nptEtXWU.md

# Introduction

AI Engineering is the process of designing and implementing AI systems using pre-trained models and existing AI tools to solve practical problems. AI Engineers focus on applying AI in real-world scenarios, improving user experiences, and automating tasks, without developing new models from scratch. They work to ensure AI systems are efficient, scalable, and can be seamlessly integrated into business applications, distinguishing their role from AI Researchers and ML Engineers, who concentrate more on creating new models or advancing AI theory.

> Learn more from the following resources:
> 
> 
> - [@video@So you want to become an AI Engineer?](https://www.youtube.com/watch?v=4RixMPF4xis)

Reviewer: ```suggestion
Learn more from the following resources:

- [@video@AI vs Machine Learning](https://www.youtube.com/watch?v=4RixMPF4xis)
```
